db/flush_job.h

```diff
@@ -81,6 +81,7 @@ class FlushJob {
   void Cancel();
   TableProperties GetTableProperties() const { return table_properties_; }
   const autovector<MemTable*>& GetMemTables() const { return mems_; }
+  const FileMetaData GetOutputFileMd() { return meta_; };
```

Comment:
Do we need to make a copy of FileMetaData? can it return `const FileMetaData&`. In fact, I don't even know whether FileMetaData is copyable.
If we only need to figure out output path, maybe a function to return output path is easier to get right.

Reply:
Changed to only return output path id.